docs/contributors/overview.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to FIRM – Overview
+
+FIRM is a **multi-technology project** that spans:
+
+- Firmware and embedded software,
+- PCB design and low-voltage electronics,
+- A browser-based web app for configuration and data viewing.
+
+These docs are the **central handbook** for new and existing contributors.
+
+---
+
+## Purpose of These Instructions
+
+The goal is to:
+
+- Give **new members** a clear, structured path to becoming productive.
+- Provide **experienced members** with a place to double-check details they might forget.
+- Capture the **rationale behind key design decisions** so we don’t lose it.
+
+This is not meant to be a generic “learn to code” or “learn electronics” guide, but a **focused resource for FIRM itself**.
```

Comment:
entire thing seems redundant, you can know all of this by just glancing through the headers, the next section literally says the same thing